cdap-ui/app/cdap/components/DataPrep/TopPanel/SchemaModal.js

```diff
@@ -70,7 +70,15 @@ export default class SchemaModal extends Component {
           fields: res
         };
 
-        let fields = getParsedSchema(tempSchema);
```

Comment:
We can't do it because we need the error from the library to be shown in the modal.